httpclient5/pom.xml

```diff
@@ -123,7 +123,7 @@
             </goals>
             <configuration>
               <url>https://publicsuffix.org/list/effective_tld_names.dat</url>
-              <outputDirectory>${project.build.outputDirectory}/mozilla</outputDirectory>
```

Comment:
@garydgregory We still need to ship a copy of this file with HttpClient artifact. Please roll back this change!